frontend/components/dashboard/validator/subset/DashboardValidatorSubsetList.vue

```diff
@@ -139,9 +143,9 @@ function mapDutyLinks(
         </BcLink>
         <template v-if="v.duty_objects?.length">
           <span class="round-brackets">
-            <span class="label">{{ mapDutyLabel(v.duty_objects) }}</span>
+            <span class="label">{{ createDutyLabels(v.duty_objects) }}</span>
             <template
-              v-for="link in mapDutyLinks(v.duty_objects)"
+              v-for="link in createDutyLinksData(v.duty_objects)"
```

Comment:
would this make sense?
```suggestion
              v-for="link in dutylinks(v.duty_objects)"
```

Reply:
It's a function so the name is an action + high context + low context. Also, it returns link data, not a link.